sagemaker-inference-recommender/sklearn-inference-recommender/skleran-inference-recommender.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    stopped = False
</pre><p>Rename this variable for accuracy. It will flip when completed/stopped/failed, so not only "stopped." How about "ended" or "concluded"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Fixed to "ended"